python/pyspark/sql/tests/test_udf.py

```diff
@@ -258,8 +258,7 @@ def test_udf_not_supported_in_join_condition(self):
         def runWithJoinType(join_type, type_string):
             with self.assertRaisesRegex(
                 AnalysisException,
-                """Using PythonUDF in join condition of join type "%s" is not supported"""
-                % type_string,
+                """Python UDF in the ON clause of a %s JOIN.""" % type_string,
```

Comment:
```suggestion
                """Python UDF in the ON clause of a "%s" JOIN.""" % type_string,
```

@cloud-fan @MaxGekk Just realized these quotes originate from the code. So I leave this to Max.